.github/workflows/test.yml

```
@@ -8,7 +8,7 @@ jobs:
   tests:
     strategy:
       matrix:
-        os: [ubuntu-18.04, macos]
+        os: [ubuntu-latest, macos-latest]
         ghc: [ghc865, ghc882]
```

It seems the official nix cache doesn’t have that GHC for macos. Or something. Let’s just drop it.
```suggestion
        ghc: [ghc882]
```

---

I thought we use 8.6.5 in the downstream, right? I'm considering updating winter and updating the dependency in downstream so we need to keep this building with 8.6.5.